cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -50,11 +53,52 @@ var (
 		"How often scale down possiblity is check")
 )
 
+var (
+	lastTimestamp = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "cluster_autoscaler",
+			Subsystem: "main",
```

Comment:
I think we don't set subsystem anywhere else. But that's probably not a big problem.